metals/src/main/scala/scala/meta/internal/metals/BloopServers.scala

```diff
@@ -138,11 +140,19 @@ final class BloopServers(
     val launcherOut = Channels.newOutputStream(clientInOutPipe.sink())
 
     val serverStarted = Promise[Unit]()
+    val bloopLogs = new OutputStream {
+      private lazy val b = new StringBuilder
+      override def write(byte: Int): Unit = byte.toChar match {
+        case c => b.append(c)
+      }
+      def logs = b.lines.toList
+    }
```

Comment:
Am I right that there are cases when bloop print error without `LF` and doesn't flush stream?
If that's true maybe add `flush` call on bloop side?

Reply:
Just checked, we always use println on that output stream.